packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.styles.ts

```diff
@@ -203,26 +203,24 @@ export const getStyles = (props: ITeachingBubbleStyleProps): ITeachingBubbleStyl
       {
         backgroundColor: palette.white,
         borderColor: palette.white,
+        color: palette.themePrimary,
         whiteSpace: 'nowrap',
         selectors: {
           // TODO: global class name usage should be converted to a button styles function once Button supports JS styling
-          [`.${classNames.buttonLabel}`]: [
-            DefaultFontStyles.medium,
-            {
-              color: palette.themePrimary
-            }
-          ],
+          [`.${classNames.buttonLabel}`]: DefaultFontStyles.medium,
```

Comment:
Minor issue, but I guess this means that font style is lost in high contrast?

Reply:
Yes, it's not apparent now because the font style is not changing in high contrast (and I wouldn't expect it to by default), but any customization would be lost.

Reply:
@khmakoto you might want to replace the usage of `DefaultFontStyles` with the values coming from the theme. Also, I wonder why do we even need these font styles here, isn’t the button already providing it’s own font styles?

Reply:
I don't know how the decision to use `DefaultFontStyles.medium` came about. Might be worth asking to design since that was in the styles before my change.

Reply:
It would worth trying removing it and inspect the styles before and after. I suppose there shouldn’t be any change as the `Button` is already having it set to font medium

Reply:
I 100% support pulling font styles from the theme.